ror.yml

```diff
@@ -24,6 +24,7 @@ rh-ror50:
     - rubygem-diff-lcs:
         macros:
           bootstrap: 1
+          _with_bootstrap: 1
```

Comment:
You are right. `_with_bootstrap: 1` that I removed should be kept.
Because ideally it should be controlled by `_with_bootstrap` variable.
As a temporary fixing, we can use "bootstrap: 1" to build it as bootstrap mode,